src/buffer/out/TextAttribute.hpp

```diff
@@ -212,7 +212,7 @@ constexpr bool operator!=(const TextAttribute& a, const TextAttribute& b) noexce
 #ifdef UNIT_TESTING
 
 #define LOG_ATTR(attr) (Log::Comment(NoThrowString().Format( \
-    L#attr L"=%s", VerifyOutputTraits<TextAttribute>::ToString(attr).GetBuffer())))
+    L## #attr L"=%s", VerifyOutputTraits<TextAttribute>::ToString(attr).GetBuffer())))

```

Comment:
`L#foo` used to turn it into a `L"foo"` string, no matter what kind of literal `foo` was.
With standard conform macros this doesn't work anymore and we have to cast `foo` to a string using `#foo` and then concatenate it with `L` which forms `L## #foo`.